docs/api.md

```diff
@@ -11,7 +11,7 @@ redirect_from:
 
 ### List metadata for all {{ site.taps.core.name }} formulae or {{ site.taps.cask.name }} casks
 
-List the `brew info --json --all` output for all current {{ site.taps.core.fullname }} formulae or {{ site.taps.cask.fullname }} casks.
+List the `brew info --json --eval-all` output for all current {{ site.taps.core.fullname }} formulae or {{ site.taps.cask.fullname }} casks.
```

Comment:
```suggestion
List the `brew info --json` output for all current {{ site.taps.core.fullname }} formulae or {{ site.taps.cask.fullname }} casks.
```

Reply:
This returns `Error: Invalid usage: this command requires a formula or cask argument`

Reply:
Yeh, I don't think this needs to be an executable command and, pretty soon, it'll require a dedicated command anyway.